### PR TITLE
[WIP] adding artifact listing based on tags

### PR DESCRIPTION
**Pull request by sbnetops**

### CODE REVIEW

File: src/polyswarm/links.py [hunk 1]
```diff
@@ -30,3 +30,18 @@ def view(ctx, sha256):
     api = ctx.obj['api']
     output = ctx.obj['output']
     output.tag_link(api.tag_link_get(sha256))
+
+
+@link.command('list', short_help='List all links with the given tags.')
+@click.option('-t', '--tag', type=click.STRING, multiple=True)
+@click.option('-f', '--family', type=click.STRING, multiple=True)
+@click.option('-g', '--or-tag', type=click.STRING, multiple=True)
+@click.option('-y', '--or-family', type=click.STRING, multiple=True)
```
Comment (mrtizmoatwork) [inline src/polyswarm/links.py]: can you add a description text for the or tag, or family args so people know what they mean?

File: src/polyswarm/links.py [hunk 2]
```diff
@@ -30,3 +30,22 @@ def view(ctx, sha256):
     api = ctx.obj['api']
     output = ctx.obj['output']
     output.tag_link(api.tag_link_get(sha256))
+
+
+@link.command('list', short_help='List all links with the given tags.')
+@click.option('-t', '--tag', type=click.STRING, multiple=True,
+              help='A tag that must be associated with the artifact.')
```
Comment (mrtizmoatwork) [inline src/polyswarm/links.py]: @sbnetops should each of these args be a "list" of "tags" or "families"? 

-y family -y family1
OR
-y family family1

Comment (sbnetops) [reply src/polyswarm/links.py]: Multiple entries, like `-y family -y family1`

Comment (mrtizmoatwork) [reply src/polyswarm/links.py]: ok. thanks

**Pull request merged**